src/core/mac/mac_frame.cpp

```diff
@@ -1518,6 +1636,63 @@ Error TxFrame::GenerateEnhAck(const RxFrame &aRxFrame, bool aIsFramePending, con
 }
 #endif // OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2
 
+#if OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE
+Error TxFrame::GenerateWakeupFrame(PanId aPanId, const Address &aDest, const Address &aSource)
+{
+    Error    error = kErrorNone;
+    uint16_t fcf   = Mac::Frame::kTypeMultipurpose | Mac::Frame::kMpFcfLongFrame | Mac::Frame::kMpFcfPanidPresent |
+                   Mac::Frame::kMpFcfSecurityEnabled | Mac::Frame::kMpFcfSequenceSuppression |
+                   Mac::Frame::kMpFcfIePresent;
```

Comment:
Can we  remove `Mac::Frame::` here? 

I guess since `TxFrame` inherits from `Mac::Frame` should have access to all these constants? (they are defined as `protected` in `Mac::Frame`).